src/behavior-considered-undefined.md

```diff
@@ -71,12 +71,16 @@ Please read the [Rustonomicon] before writing unsafe code.
   * A `!` (all values are invalid for this type).
   * An integer (`i*`/`u*`), floating point value (`f*`), or raw pointer obtained
     from [uninitialized memory][undef], or uninitialized memory in a `str`.
-  * A reference or `Box<T>` that is [dangling], misaligned, or points to an invalid value.
-  * Invalid metadata in a wide reference, `Box<T>`, or raw pointer:
-    * `dyn Trait` metadata is invalid if it is not a pointer to a vtable for
-      `Trait` that matches the actual dynamic trait the pointer or reference points to.
```

Comment:
"that matches the actual dynamic trait" was clearly nonsense. Also this is now moved to the "points to an invalid" point since it's not about the metadata, it's about using the metadata to keep going recursively through the reference.